lib/compiler.js

```diff
@@ -85,6 +85,10 @@ module.exports = function compile(options) {
   var modelInstructions = buildAllModelInstructions(
     modelsRootDir, modelsConfig, modelSources);
 
+  var mixinDirs = options.mixinDirs || modelsMeta.mixins || ['./mixins'];
```

Comment:
Also `modelsMeta.mixins` should be used to initialise `mixinSources`, not `mixinDirs`.

```js
var mixinDirs = options.mixinDirs || [];
var mixinSources = options.mixinSources || modelsMeta.mixins || ['./mixins'];
```